Jenkinsfile

```diff
@@ -80,6 +80,24 @@ pipeline {
             '''
           }
         }
+        stage('Nodejs client tests') {
+          agent { label 'medium' }
+          steps {
+            checkout scm
+            sh '''
+              export HOME=$(pwd)
+              export LANG=en_US.UTF-8
+              test -d env && rm -rf env
+              python3.8 -m venv env
+              . env/bin/activate
+              python3.8 -m pip install -U cr8
+              curl -sL https://deb.nodesource.com/setup_13.x | sudo -E bash -
```

Comment:
We shouldn't have any jobs that try to install packages via `sudo`. Either we've to switch to using a docker image, or make sure the required packages are present on the Jenkins slaves